### PR TITLE
Add_clip fusion

### PR DESCRIPTION
**Pull request by kahmed10**

### CODE REVIEW

File: src/onnx/CMakeLists.txt [hunk 1]
```diff
@@ -32,11 +27,4 @@ add_executable(cifar10 cifar10.cpp)
 rocm_clang_tidy_check(cifar10)
 target_link_libraries(cifar10 migraphx_cpu migraphx_gpu migraphx_onnx)
 
-add_executable(verify_onnx verify_onnx.cpp)
-rocm_clang_tidy_check(verify_onnx)
-target_link_libraries(verify_onnx migraphx_onnx migraphx_cpu migraphx_gpu)
-
-add_executable(perf_onnx perf_onnx.cpp)
-rocm_clang_tidy_check(perf_onnx)
-target_link_libraries(perf_onnx migraphx_onnx migraphx_cpu migraphx_gpu)
```
Comment (pfultz2) [inline src/onnx/CMakeLists.txt]: Why are these changes here?

File: src/tf/CMakeLists.txt [hunk 2]
```diff
@@ -29,17 +29,3 @@ target_link_libraries(migraphx_tf PUBLIC migraphx)
...
-
-if(MIGRAPHX_ENABLE_GPU)
-add_executable(verify_tf verify_tf.cpp)
-rocm_clang_tidy_check(verify_tf)
-target_link_libraries(verify_tf migraphx_tf migraphx_cpu migraphx_gpu)
-
-add_executable(perf_tf perf_tf.cpp)
-rocm_clang_tidy_check(perf_tf)
-target_link_libraries(perf_tf migraphx_tf migraphx_cpu migraphx_gpu)
-endif()
```
Comment (pfultz2) [inline src/tf/CMakeLists.txt]: These changes should be a separate PR.

Comment (kahmed10) [reply src/tf/CMakeLists.txt]: fixed

File: src/onnx/CMakeLists.txt (hunk 1 above)
Comment (kahmed10) [reply src/onnx/CMakeLists.txt]: fixed
